docs/book/src/tasks/kubeadm-control-plane.md

```diff
@@ -39,7 +39,10 @@ but is the final step in fully upgrading a Cluster API managed cluster.
 
 It is recommended to manage workload machines with one or more `MachineDeployment`s. `MachineDeployment`s will
 transparently manage `MachineSet`s and `Machine`s to allow for a seamless scaling experience. A modification to the
-`MachineDeployment`s spec will begin a rolling update of the workload machines.
+`MachineDeployment`s spec will begin a rolling update of the workload machines. As described in 
+[this section on changing a `MachineTemplate`](./change-machine-template.md), infrastructure machine 
+templates are immutable and should not be modified directly; follow the instructions there for 
```

Comment:
Given that infrastructure provider might relax the immutability, here I suggest to simplify an simply link the other page. e.g.

follow [these instruction](link) for changing the template for an existing `MachineDeployment`

Reply:
Changes incorporated into af55bb4.